src/ophyd_async/epics/signal/signal.py

```diff
@@ -58,6 +58,28 @@ def epics_signal_rw(
     return SignalRW(backend)
 
 
+def epics_signal_rw_rbv(
```

Comment:
I think we still want it to be called `epics_signal_rw`

Reply:
nope, this is the additional function to specifically do `prefix`, `prefix + "_RBV"` PV pairs

Reply:
nope, we already have one of these, this is a second function for when we have a `prefix`, `prefix + "_RBV"` record pair